server-adapters/resteasy-netty4-cdi/pom.xml

```diff
@@ -14,7 +14,7 @@
     <description/>
     <properties>
         <dep.arq.weld.se>2.0.0.Final</dep.arq.weld.se>
-        <dep.arq.version>1.1.15.Final</dep.arq.version>
+       <dep.arq.version>1.4.1.Final</dep.arq.version>
```

Comment:
uh, just spotted this. Is there a reason why we can't use the 'dep.arquillian-bom.version' from the resteasy-dependencies-bom/pom.xml? (to avoid duplicating the property)

Reply:
rebase .. complete
version ref change .. complete